assembl/static2/js/app/components/administration/saveButton.jsx

```diff
@@ -0,0 +1,192 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { gql, graphql, withApollo, compose } from 'react-apollo';
+import { Button } from 'react-bootstrap';
+import { Translate, I18n } from 'react-redux-i18n';
+import { displayAlert } from '../../utils/utilityManager';
+
+const GetThematics = gql`
+{
+  thematics(identifier:"survey") {
+    id,
+    titleEntries {
+      localeCode,
+      value
+    },
+    imgUrl,
+    video {
+      title,
+      description,
+      htmlCode
+    }
+    questions {
+      titleEntries {
+        localeCode,
+        value
+      }
+    }
+  }
+}
+`;
+
+const createLanguageEntries = (titles) => {
```

Comment:
We'll need it elsewhere. We can put it in `utils` but there are many things inside. Maybe we can create a helper dedicated to Apollo mutations...